perception/image_projection_based_fusion/docs/roi-detected-object-fusion.md

```diff
@@ -6,7 +6,10 @@ The `roi_detected_object_fusion` is a package to overwrite labels of detected ob
 
 ## Inner-workings / Algorithms
 
-The detected objects are projected onto image planes, and then if the ROIs of detected objects (3D ROIs) and from a 2D detector (2D ROIs) are overlapped, the labels of detected objects are overwritten with that of 2D ROIs. Intersection over Union (IoU) is used to determine if there are overlaps between them.
+The algorithm is shown below and see `Parameters` for the meaning of each parameter :
+
+1. If `existence_probability` of detected objects is greater than the threshold, they passthrough without doing anything.
+2. The rest of detected objects are projected onto image planes, and if ROIs are overlapped with them, they are output as fused objects. Intersection over Union (IoU) is used to determine if there are overlaps between them.
 
 The `DetectedObject` has three shape and the polygon vertices of a object are as below:
```

Comment:
The wording makes it very difficult to read/understand.

The `DetectedObject` has three possible `shape` choices/implementations, where the polygon's vertices for each case are defined as follows:

Reply:
updated at [886ce8a](https://github.com/autowarefoundation/autoware.universe/pull/2178/commits/886ce8a894c29e8fd0df34f56bc743980583a86b).